PhysicsTools/Heppy/test/crab/heppy_config.py

```diff
@@ -1 +0,0 @@
-../example_autofill.py
```

Comment:
Note to self: No idea what happened there. This should be removed from the commit.

Reply:
Solution: was a symlink, turned into a copy due to the scripting. Will be fixed soon.